frontend/src/components/PageTitle.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { useRoute } from 'vue-router'
+import { selectedApplicationDisplayText } from '@/services/ApplicationState'
+
+const props = defineProps<{
+  displaySelectedApplication?: boolean
+}>()
+const displaySelectedApplicationText = props.displaySelectedApplication || true // default to true.
+
+</script>
+
+<template>
+  
+  <div class="title" v-if="displaySelectedApplicationText">{{selectedApplicationDisplayText}}</div>
```

Comment:
I'm personally not a fan of the format of the title, but I like the design of making it a separate component and using the Router meta - that's a neat trick!